flutter/example/README.md

```diff
@@ -1,16 +1,41 @@
-# sentry_flutter_example
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br/>
+</p>
 
-Demonstrates how to use the sentry_flutter plugin.
+Sentry Example for Flutter
+===========
 
-## Getting Started
+This is Flutter app that crashes intentionally in various ways. It demonstrates
```

Comment:
```suggestion
This is a Flutter App. that crashes intentionally in different ways. It demonstrates
```

Reply:
I believe `various` is the correct term here because `various` kinda means `many different` but I'm not a native speaker so 🤷‍♂️